core/kernel.rbs

```diff
@@ -1301,10 +1332,7 @@ module Kernel : BasicObject
   #
   # See also Random.rand.
   #
-  def self?.rand: () -> Float
-                | (Integer arg0) -> Integer
-                | (::Range[Integer] arg0) -> Integer
-                | (::Range[Float] arg0) -> Float
```

Comment:
Have you finished `self?.rand`? My IRB accepts both `0.0` and `0..0`.

Reply:
No I havent, it's one of the remaining functions left. It's really weird.